### ISSUE TITLE
[ERR_INVALID_ARG_TYPE]: The "from" argument must be of type string.

### ISSUE THREAD

**Issue by dudo** [autifyhq/parcel-reporter-bundle-manifest#243]
```sh
TypeError [ERR_INVALID_ARG_TYPE]: The "from" argument must be of type string. Received undefined
    at validateString (internal/validators.js:124:11)
    at Object.relative (path.js:1167:5)
    at Object.report (/usr/src/app/node_modules/parcel-reporter-bundle-manifest/lib/BundleManifestReporter.js:1:577)
    at ReporterRunner.report (/usr/src/app/node_modules/@parcel/core/lib/ReporterRunner.js:105:33)
    at async Parcel._build (/usr/src/app/node_modules/@parcel/core/lib/Parcel.js:449:7)
    at async Parcel._startNextBuild (/usr/src/app/node_modules/@parcel/core/lib/Parcel.js:301:24)
    at async PromiseQueue._runFn (/usr/src/app/node_modules/@parcel/utils/lib/PromiseQueue.js:88:7)
    at async PromiseQueue._next (/usr/src/app/node_modules/@parcel/utils/lib/PromiseQueue.js:75:5) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

Just ran an npm update, and parcel doesn't want to build anymore. I flushed `.parcel-cache`, `dist`, and `node_modules`

```js
"parcel": "^2.0.0-rc.0",
"parcel-reporter-bundle-manifest": "^0.3.3"
```

**Comment by jmacedoit** [comment]
I'm also facing with this issue.

**Comment by joshuabaker** [comment]
This issue is caused by this line...

https://github.com/autifyhq/parcel-reporter-bundle-manifest/blob/ec4cd8e800a5343faa7ed02d301b1f1e14ad1bac/src/BundleManifestReporter.ts#L50

Both `options.entryRoot` and `options.rootDir` are `undefined`.

Simply falling back to an empty string gets the `parcel-manifest.json` file to write, however, all of the hashes are borked (probably another issue).

**Comment by joshuabaker** [comment]
Already sorted in #237.

**Comment by lukyth** [comment]
This should be resolved with `v1.0.0` as the plugin is supporting Parcel 2.0.0 now. Special thanks to @joshuabaker and @akeylimepie!

Please try using that version and let me know if the issue still occurs.

```sh
npm install --save-dev parcel-reporter-bundle-manifest@^1.0.0
```
